Makefile

```diff
@@ -0,0 +1,8 @@
+build:
+	go build -o sail .
+
+deps:
+	go get ./...
+
+install: deps build
```

Comment:
Install rules usually only move and don't depend on build. As this Makefile is right now, if you run `make install` it will run deps, then build, then install even if you've already done `make build`.

The `install` rule should not depend on deps or build in my opinion.

Reply:
should there be a separate step or something for an all in one?

Reply:
Maybe `all`? Not sure.